frontend-react/src/utils/UserUtils.ts

```diff
@@ -0,0 +1,20 @@
+import { OktaAuth } from "@okta/okta-auth-js";
+import { GLOBAL_STORAGE_KEYS } from "../components/GlobalContextProvider"
+import { oktaAuthConfig } from "../oktaConfig";
+
+const OKTA_AUTH = new OktaAuth(oktaAuthConfig);
+
+function clearGlobalContext() {
+    for (let key in localStorage) {
+        if (key.includes(GLOBAL_STORAGE_KEYS.GLOBAL_BASE)) localStorage.removeItem(key)
```

Comment:
We should probably always use `{ }` for `if` blocks even if they are one line. I think when we enforce a style guide, this will be an error.

Reply:
Fixed in new commit! :+1: